src/istgt_lu_disk.c

```diff
@@ -5736,7 +5751,9 @@ istgt_lu_disk_lbwrite(ISTGT_LU_DISK *spec, CONN_Ptr conn, ISTGT_LU_CMD_Ptr lu_cm
 	if (markedForReturn == 1) {
 		ISTGT_ERRLOG("c#%d Error in entering block call", conn->id);
 		goto freeiovcnt;
-	}
+	} else if (markedForReturn == 2)
+		return 0;
```

Comment:
goto freeiovcnt is required.. add logs for markedForReturn == 2 case

Reply:
Ok.